pyproject.toml

```diff
@@ -117,10 +117,6 @@ log_cli_level = "INFO"
 xfail_strict = true
 filterwarnings = ["error",
     'ignore:__package__ != __spec__.parent:ImportWarning',
```

Comment:
It might be possible that this warning can also be removed. Not 100% sure though.